static/app/components/demoSandboxButton.tsx

```diff
@@ -0,0 +1,94 @@
+import Button from 'sentry/components/button';
+import {Organization} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  /**
+   * The deep link scenario
+   */
+  scenario:
+    | 'performance'
+    | 'releases'
+    | 'alerts'
+    | 'discover'
+    | 'dashboards'
+    | 'projects'
+    | 'oneDiscoverQuery'
+    | 'oneIssue'
+    | 'oneBreadcrumb'
+    | 'oneStackTrace'
+    | 'oneTransaction'
+    | 'oneWebVitals'
+    | 'oneTransactionSummary'
+    | 'oneRelease';
+  organization: Organization;
+  buttonText: string;
+  /**
+   * Which project we should link to in the sandbox
+   */
+  projectSlug?: 'react' | 'python' | 'ios' | 'android' | 'react-native';
+  /**
+   * Matching on the error type or title
+   */
+  errorType?: string;
+
+  buttonProps?: Partial<React.ComponentProps<typeof Button>>;
```

Comment:
Instead of doing this why don't we have the Props intersect with `React.ComponentProps<typeof Button>` (you don't need partial since almost all button props aren't required).

Then just spread `...props` into the button.